Program.cs

```diff
@@ -121,8 +121,9 @@
 );
 if (builder.Environment.IsDevelopment())
 {
-    var mvc = builder.Services.AddRazorPages();
-    mvc.AddRazorRuntimeCompilation();
+    // handy when developing frontend stuff. Can cause wonkiness when server side rendering pages
+    // var mvc = builder.Services.AddRazorPages();
+    // mvc.AddRazorRuntimeCompilation();
```

Comment:
It would be best to have it commented out here?

Reply:
Ya its a bit wonky for server side rendering stuff like when we pull data from the DB when we render a new page for example. Super nice to have for frontend stuff but its probably a good idea to leave it disabled unless we need it